.env.example

```
@@ -76,3 +76,9 @@ STRIPE_DONATION_PRICING_TABLE_ID=
 STRIPE_DONATION_PRICING_TABLE_KEY=
 STRIPE_CUSTOMER_PORTAL_URL=
 STRIPE_ONE_TIME_DONATION_URL=
+
+# Minecraft Server - player-facing connection info (shown in the verification UI)
+MINECRAFT_SERVER_NAME="Lighthouse MC"
+MINECRAFT_SERVER_HOST=play.lighthousemc.net
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINECRAFT_SERVER_HOST key should go before the MINECRAFT_SERVER_NAME key

(UnorderedKey)